package.json

```diff
@@ -21,6 +21,7 @@
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
+    "semantic-ui-react": "^1.2.1",
```

Comment:
this shouldn't be here right...

Reply:
We need it for our button divs

Reply:
a) I don't think you do
b) if you do - let's add it when you add those buttons.

Either way, it shouldn't be part of this PR